.github/workflows/test_workflow_scripts/golang/echo_sql/golang-docker-macos.sh

```diff
@@ -97,9 +95,11 @@ fi
 
 if grep "WARNING: DATA RACE" "${test_container}.txt"; then
     echo "Race condition detected in test, stopping pipeline..."
+    cat "${test_container}.txt"
```

Comment:
Instead of directly piping into the file, I am using ` 2>&1 | tee` so dont need to cat again in case of failure or race warning